docs/root/intro/arch_overview/upstream/outlier.rst

```diff
@@ -145,6 +145,8 @@ Most configuration items, namely
 types of errors, but :ref:`outlier_detection.enforcing_success_rate<envoy_api_field_cluster.OutlierDetection.enforcing_success_rate>` applies
 to externally originated errors only and :ref:`outlier_detection.enforcing_local_origin_success_rate<envoy_api_field_cluster.OutlierDetection.enforcing_local_origin_success_rate>`  applies to locally originated errors only.
 
+For gRPC requests, the outlier detection will use the HTTP status mapped from the [grpc-status](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#responses) response header.
```

Comment:
This is not the way external links work in sphinx/RST. You can view the rendered docs in the CI artifacts or you can build the docs locally using `docs/build.sh` to view them if that helps.

Also, nit, can you potentially put this under a gRPC sub-section just to draw attention to it?

Reply:
Done.